assets/queries/openAPI/server_url_not_absolute/query.rego

```diff
@@ -71,6 +71,7 @@ CxPolicy[result] {
 	openapi_lib.check_openapi(doc) != "undefined"
 
 	url := doc.components.responses[r].links[l].server.url
+	openapi_lib.content_allowed("", r)
```

Comment:
I think this does not makes sense, since components does not need response code, as you can see [here](https://swagger.io/specification/#components-object)